bugbug/models/exception.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+class CustomException(Exception):
```

Comment:
Let's define this directly in bugbug/db.py, and let's call it "LastModifiedNotAvailable"

Reply:
Doing this should also avoid the circular import problem that you're hitting.

Reply:
implemented the changes mentioned